tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -54,23 +101,26 @@ case "$DATABASE_TYPE" in
     fi
     ;;
 esac
-
-# Initialize database if needed
+loggood "checking of database success"
+# Initialize database if needed
 if [ "$INIT_DATABASE" = true ] ; then
+    log "Initialise Database"
     cd /tracim/backend/
-    tracimcli db init -c /etc/tracim/development.ini
+    tracimcli_as_user "db init"
```

Comment:
`tracimcli` command is no more supported ?

Which development.ini file is used here ?

Reply:
`tracimcli_as_user` is a function defined at the beginning of the script. It still uses `tracimcli`.